src/state/reducers/workOrdersReducer.js

```diff
@@ -0,0 +1,23 @@
+import { RENDER_SOMETHING, GET_WORKORDERS, GET_WORKORDER, SUBMIT_WORKORDER, MARK_URGENT, MARK_NORMAL } from "../actions/allWo";
+
+const initialState = {};
+
+export const workOrdersReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case RENDER_SOMETHING:
+            console.log("12:", action.payload)
+            return {...state, workorders: action.payload};
+        case GET_WORKORDERS:
+            return { ...state, workorders: action.payload, loading: false};
+        case GET_WORKORDER:
+            return {...state, workorder: action.payload, loading: false};
+        case SUBMIT_WORKORDER:
+            return [...state, action.payload]
+        case MARK_URGENT:
+            return [...state.map(wo => wo.id === action.payload ? {...wo, urgent: !wo.urgent } : wo)];
```

Comment:
Work Order objects don't have an `urgent` property, unless you're adding one for your internal use. They have a `priority` property which uses an integer for its value:
1. == { name: 'Critical', color: '#F03911' },
2. == { name: 'High', color: '#F7931B' },
3. == { name: 'Medium', color: '#F4F71B' },
4. == { name: 'Low', color: '#5AF71B' },